.github/CODEOWNERS

```diff
@@ -0,0 +1,86 @@
+# Lists some code owners.
+#
+# A codeowner just oversees some part of the codebase. If an owned file is changed then the
+# corresponding codeowner receives a review request. An approval of the codeowner might be
+# required for merging a PR (depends on repository settings).
+#
+# For details about syntax, see:
+# https://help.github.com/en/articles/about-code-owners
+# But here are some important notes:
+#
+# - Glob syntax is git-like, e.g. `/core` means the core directory in the root, unlike `core`
+#   which can be everywhere.
+# - Multiple owners are supported.
+# - Either handle (e.g, @github_user or @github/team) or email can be used. Keep in mind,
+#   that handles might work better because they are more recognizable on GitHub,
+#   eyou can use them for mentioning unlike an email.
+# - The latest matching rule, if multiple, takes precedence.
+
+# Top levels
+/docker @paritytech/ci @paritytech/release-engineering
+/prdoc @paritytech/release-engineering
+/docs @paritytech/docs-audit
+/substrate @paritytech/SubstrateTeamLeads
+/polkadot @paritytech/polkadot-review
+/cumulus @paritytech/cumulus-locks-review
```

Comment:
Is this the catch all for `cumulus`? Normally the locks review is only required for runtime files.  
Mabye use `core-devs` or so.

Reply:
Yes, I wasn't 100% of those but I am sure we want at least a value. 
There is no **team** (in GH) for cumulus other than this one. I am totally fine with `core-devs` instead and I pinged @gilescope to see if we could provide more specific patterns below this catch-all to route review more accurately.

Reply:
I **added** `core-devs` for now.
Keep in mind those are catch-all so we can add more specific patterns below that are more targetted in terms of content and people.